conn_test.go

```diff
@@ -560,6 +560,139 @@ func TestClientCertificate(t *testing.T) {
 	}
 }
 
+func TestExtendedMasterSecret(t *testing.T) {
+	t.Parallel()
```

Comment:
Did not know about this! Should we set this for all our tests? How much of an improvement do you see locally?

Reply:
I didn't either, I actually just took it from the `TestClientCertificate` test 😅